scvelo/tools/transition_matrix.py

```diff
@@ -36,6 +36,9 @@ def transition_matrix(
     from the velocity graph :math:`\\pi_{ij}`, with row-normalization :math:`z_i` and
     kernel width :math:`\\sigma` (scale parameter :math:`\\lambda = \\sigma^{-1}`).
 
+    Other option would be to use :func:`cellrank.tl.transition_matrix`
+    which can also take into account velocity uncertainty.
+
```

Comment:
Again, "An alternative option is given by XXX which takes into account uncertainty in the velocity vectors when computing transition probabilities.  "